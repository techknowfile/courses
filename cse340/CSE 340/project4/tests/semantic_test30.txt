TYPE
   q, r, s: BOOLEAN;
   u, v : LONG;
VAR
   a : r;
   b : q;
   c : u;
   d : v;
{
    WHILE a <= 2.4 {
         c = c + d;
    }
}

