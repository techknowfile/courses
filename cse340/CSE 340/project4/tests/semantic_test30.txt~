TYPE
   q, r, s: BOOLEAN;
   u, v : LONG;
VAR
   a : r;
   b : q;
   c : u;
   d : v;
{
    WHILE 1 <= x {
         c = c + d;
    }
}

