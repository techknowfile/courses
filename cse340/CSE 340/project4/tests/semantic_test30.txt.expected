TYPE MISMATCH 10 C3
